butane/Cargo.toml

```diff
@@ -26,17 +26,14 @@ uuid = ["butane_core/uuid", "butane_codegen/uuid"]
 
 [dependencies]
 butane_codegen = { path = "../butane_codegen", version = "0.5" }
-butane_core = { path = "../butane_core", version = "0.5" }
+butane_core = { path = "../butane_core", version = "0.5", features = ["sqlite"] }
```

Comment:
this shouldnt be necessary.

maybe you need sqlite for the tests?  if so, you may be able to add it as a `dev-dependencies` by using https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#renaming-dependencies-in-cargotoml

Reply:
Reverted.